index.html

```diff
@@ -247,6 +257,58 @@ <h3>Skills</h3>
                   </div>
                 </div>
               </section>
-             </main>
-            </body>
-          </html>
+
+              <!--Form section-->
+    <section id="form-section">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form
+        action="#"
+        method="POST"
+        id="contact-form"
+      >
+        <fieldset>
+          <input
+            class="inputs"
+            type="text"
+            name="full-name"
+            id="fullname"
+            maxlength="30"
+            placeholder="Full name"
+            required
+          />
+          <input
+            class="inputs"
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email address"
+            required
+          />
+
+          <textarea
+            name="comments"
+            id="comments"
+            maxlength="500"
+            placeholder="Enter text here"
+          ></textarea>
+        </fieldset>
+        <input type="submit" id="submit" name="submit" value="Get in touch" />
+        <small class="small hidden show"
+          >Please, enter valid Email: lower case only.</small
+        >
+      </form>
+    </section>
+    <!--Footer-->
+    <footer id="form-icons">
+      <img src="icons/vector4.png" alt="github icon" />
+      <img src="icons/Disabled.png" alt="linkedin icon" />
+      <img class="gap" src="icons/vector1.png" alt="twitter" />
+      <img class="gap" src="icons/vector2.png" alt="hand" />
+      <img src="icons/vector3.png" alt="medium" />
+     </footer>
```

Comment:
- [ ] Please try to get rid of the black space in the footer to match the Figma design

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/161102187-b5033de9-75a5-4851-a8e5-0b4119d77a56.png) | ![image](https://user-images.githubusercontent.com/24703171/161102347-927c7cb1-43f7-4f86-9f3c-86b8498f991d.png) |

- [ ] Please wrap the images with a `ul` tag and each image with a `li` tag, so it will be like:
```
<ul>
   <li><img src="icons/vector4.png" alt="github icon" /></li>
   ...........................................
</ul>
```